rllib/execution/multi_gpu_learner_thread.py

```diff
@@ -147,17 +148,25 @@ def step(self) -> None:
             buffer_idx, released = self.ready_tower_stacks_buffer.get()
 
         with self.grad_timer:
-            fetches = self.policy.learn_on_loaded_batch(
+            learner_info_builder = LearnerInfoBuilder(
+                num_devices=len(self.policy.devices))
+            default_policy_results = self.policy.learn_on_loaded_batch(
                 offset=0, buffer_index=buffer_idx)
+            learner_info_builder.add_learn_on_batch_results(
```

Comment:
maybe just add a comment here pointing out that "multi-gpu for multi-agent" is not supported yet.

Reply:
Added a comment to each location, where we use the new LearnerInfoBuilder explaining that it unifies results dict structures across the different setups (multi-GPU, multi-agent, tf/torch, etc..).